### DISCUSSION TITLE
Q about build date in `setuptools` version

### DISCUSSION THREAD

**Discussion by kloczek** [pypa/setuptools#3041] [General]
I realised that by default `setuptools` is using in setup.cfg `tag_date = 1`.
Result is that to the version string is added current date like below
```console
[tkloczko@ss-desktop setuptools-60.5.4]$ pip list | grep setuptools
setuptools                    60.5.4.post20220117
```
Q: Is it any method to change that settiong to `tag_date = 0` from command line on pep517 based build (like `python3 -sBm build -w --no-isolation`) or only way is just patch that before start build?
Maybe it is some env variable wchich allows to chenge that on build time? 🤔

**Comment by kloczek** [comment]
Just tested that `python3 -sBm build -w --no-isolation -C=--tag_build= -C=--tag_date=0` does not work.

**Reply by webknjaz** [reply]
That syntax refers to passing backend-specific config settings which is not implemented at the moment.

**Comment by abravalheri** [comment]
This is currently not supported, see https://github.com/pypa/setuptools/issues/2955 an https://github.com/pypa/setuptools/issues/2491

PRs are more than welcome 😄
